### DISCUSSION TITLE
Updating progress task position when tasks are removed

### DISCUSSION THREAD

**Discussion by honno** [Textualize/rich#430] [Q&A]
I was wondering if it is possible for the task table to move "down" when a task is removed, i.e. I'm interested in "removing" the empty lines that occupy the space between the terminal's "bottom" and the existing tasks. I've been looking at the internals but currently still have no idea on this haha (cursor positioning is blowing my mind). Would be extremely grateful for just a "this is possible/not possible".

**Comment by willmcgugan** [comment]
I don't _think_ that's possible. The would be adjusting the scroll position which an app can't do explicitly. 

Best I can do is "probably not possible" ;)

**Reply by honno** [reply]
Thanks for responding, I now see how convoluted this would be lol.
